docs/cisco.iosxr.iosxr_bgp_module.rst

```diff
@@ -14,6 +14,13 @@ Version added: 1.0.0
    :local:
    :depth: 1
 
+DEPRECATED
+----------
+:Removed in collection release after 2023-01-29
```

Comment:
This is probably < 2 years given today's date. Shouldn't it be `2024-01-01`?

Reply:
the date is right as we have released this module in jan 2021 so removal date is jan 2023 please check deprecation dates of others platforms also. i have created separate pr to fix this issue as current pr is failing with missing changelog fragment error. plz review this  https://github.com/ansible-collections/cisco.iosxr/pull/184